src/hyp3_isce2/insar_tops.py

```diff
@@ -59,7 +59,14 @@ def insar_tops(
     roi = slc.get_dem_bounds(ref_dir, sec_dir)
     log.info(f'DEM ROI: {roi}')
 
-    dem_path = download_dem_for_isce2(roi, dem_name='glo_30', dem_dir=dem_dir, buffer=0)
+    dem_dir.mkdir(exist_ok=True, parents=True)
+    dem_path = dem_dir / 'full_res.dem.wgs84'
+    download_dem_for_isce2(roi, dem_path=dem_path, pixel_size=30.0)
+    geocode_dem_path = dem_path
+    if range_looks == 5:
+        geocode_dem_path = dem_path = dem_dir / 'full_res_geocode.dem.wgs84'
```

Comment:
Is `geocode_dem_path` supposed to equal `dem_path`? Or is this an if/else based on range looks?

Reply:
But yeah I see the weirdness in line 67. I'll change it.

Reply:
Honestly, I'm surprised you can even do that type of assignment.